docs/security_considerations.md

```diff
@@ -14,9 +14,19 @@ docker run --rm -d --user $(id -u):$(id -g) --read-only --tmpfs /tmp -v ${PWD}/m
 --model_path /models/resnet/ --model_name resnet
 
 ``` 
+---
 OpenVINO Model Server currently does not provide access restrictions and traffic encryption on gRPC and REST API endpoints. The endpoints can be secured using network settings like docker network settings or network firewall on the host. The recommended configuration is to place OpenVINO Model Server behind any reverse proxy component or load balancer, which provides traffic encryption and user authorization.
 
 See also:
 - [Securing OVMS with NGINX](../extras/nginx-mtls-auth/README.md)
 - [Securing models with OVSA](https://docs.openvino.ai/2023.1/ovsa_get_started.html)
 
+---
+
+OpenVINO Model Server has a set mechanisms preventing denial of service attacks from the client applications. They include the following:
+- setting the number of inference execution streams which can limit the number of parallel inference calls in progress in each model. It can be tuned with `INFER_STREAM` or `PERFORMANCE_HINT` plugin config.
```

Comment:
INFER_STREAM 
```suggestion
- setting the number of inference execution streams which can limit the number of parallel inference calls in progress in each model. It can be tuned with `NUM_STREAMS` or `PERFORMANCE_HINT` plugin config.
```